web/src/layout/Header/index.tsx

```diff
@@ -27,8 +32,11 @@ export const PopupContainer = styled.div`
 const Header: React.FC = () => {
   return (
     <Container>
-      <DesktopHeader />
-      <MobileHeader />
+      {process.env.REACT_APP_DEPLOYMENT === "testnet" && <TestnetBanner />}
```

Comment:
sorry I forgot to mention it in the previous comment, let's switch this to the ternary operator too
```suggestion
      {process.env.REACT_APP_DEPLOYMENT === "testnet" ? <TestnetBanner /> : null}
```